internal/storage/v2/grpc/tracereader_test.go

```diff
@@ -17,24 +17,41 @@ import (
 	"github.com/jaegertracing/jaeger/proto-gen/storage/v2"
 )
 
+type getServicesRepsonse struct {
```

Comment:
I think these types are redundant, you can have a shared `err` field, which will work fine as long as you don't reuse the same testServer across different tests